test/executables/react16_upgrade_test.dart

```diff
@@ -71,25 +70,15 @@ ProcessResult runUpgrade({String onDirectory}) {
   return result;
 }
 
-ProcessResult runUpgradeWithFakeDartProject(
```

Comment:
Inlined this so I wouldn't have to change `pubspecCreator` to a list another time haha